images/win/scripts/Installers/Validate-Git.ps1

```diff
@@ -25,25 +25,41 @@ if( $(git-lfs version) -match  'git-lfs\/(?<version>.*) \(Git.*' )
    $gitLfsVersion = $Matches.version
 }
 
-# Adding description of the software to Markdown
-$SoftwareName = "Git"
+if($(hub version | Select-String "hub version") -match 'hub version (?<version>.*)')
+{
+    $hubVersion = $Matches.version
+}
 
-$Description = @"
+# Adding description of the software to Markdown
+$GitSoftware = @(
+    [pscustomobject]@{
+        SoftwareName = "Git";
+        Description = @"
 _Version:_ $gitVersion<br/>
 _Environment:_
 * PATH: contains location of git.exe
 "@
-
-Add-SoftwareDetailsToMarkdown -SoftwareName $SoftwareName -DescriptionMarkdown $Description
-
-# Adding description of the software to Markdown
-$SoftwareName = "Git Large File Storage (LFS)"
-
-$Description = @"
+    },
+    [pscustomobject]@{
+        SoftwareName = "Git Large File Storage (LFS)";
+        Description = @"
 _Version:_ $gitLfsVersion<br/>
 _Environment:_
 * PATH: contains location of git-lfs.exe
 * GIT_LFS_PATH: location of git-lfs.exe
```

Comment:
Where do we set the GIT_LFS_PATH variable?

Reply:
I didn't find setting this variable somewhere in install scripts and in VMs, but this variable may used in some builds. I suggest to leave it here

Reply:
Please provide an example output from Windows Server agents where the `GIT_LFS_PATH`  variable is defined.